test/test-persistent.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+set -e
+
+TEST_USER=test
+TEST_PASSWORD=test
+
+oc delete all,secrets,sa,templates,configmaps,daemonsets,clusterroles,rolebindings,serviceaccounts --selector=template=infinispan-persistent || true
+oc delete template infinispan-persistent || true
+
+
+oc create -f templates/infinispan-persistent.json
+
+
+oc new-app infinispan-persistent \
+  -p APPLICATION_USER=$TEST_USER \
+  -p APPLICATION_PASSWORD=$TEST_PASSWORD
+
+
+# TODO http should be exposed by default
+oc expose svc/infinispan-persistent-app-http
+
+
+isPodReady() {
+  oc get pod -l application=infinispan-persistent-app -o jsonpath="{.items[0].status.containerStatuses[0].ready}"
+}
+
+getPodStatus() {
+  oc get pod -l application=infinispan-persistent-app -o jsonpath="{.items[0].status.containerStatuses[0].state}"
+}
+
+waitForReady() {
+  ready="false"
+  while [ "$ready" != "true" ];
+  do
+    ready=$(isPodReady)
+    status=$(getPodStatus)
+    echo "Pod: ready=${ready},status=${status}"
+    sleep 10
+  done
+}
+
+waitForReady
+
+#getRouteHost() {
```

Comment:
Why is this commented?

Reply:
Because the persistent test does not yet work.